src/adapters/MeetingsJSONAdapter.js

```diff
@@ -185,6 +188,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
         // Add a video stream as if it were a remote meeting
         if (meeting.remoteVideo instanceof MediaStream) {
           meeting.remoteVideo = this.getVideoStream();
+          meeting.state = MeetingState.JOINED;
```

Comment:
this should not be needed, as `joinMeeting()` takes care of it. And already existing meetings in the datasource should have their `state`s set to JOINED from the start.